### ISSUE TITLE
App update dialog doesn't peoperly display buttons

### ISSUE THREAD

**Issue by MariusAlbrecht** [KotatsuApp/Kotatsu#880]
### Brief summary

buttons get displayed one per line in a scrollable view, like this:

https://github.com/KotatsuApp/Kotatsu/assets/64852250/8434c91f-94d1-4421-adf9-d34b5c36c4ef

I expect them to all get displayed _without_ the need to scroll (and optimally, side-by-side). That also seems to be the intent the material design has:
[https://m2.material.io/components/dialogs/android#alert-dialog](https://m2.material.io/components/dialogs/android#alert-dialog) (tho this example is v2, newer ones also seem to at least suggest keeping all action buttons in view)

I think this is an actual issue because I do not expect a scrollable view with options visible only after scrolling. That essentially hides the neutral button (and partially the negative one) which shouldn't be the case imo.

### Steps to reproduce

get the update dialog

### Kotatsu version

6.8.3

### Android version

12

### Device

OnePlus 7 Pro

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] If this is an issue with a parser, I should be opening an issue in the [parsers repository](https://github.com/KotatsuApp/kotatsu-parsers/issues/new/choose).

**Comment by MariusAlbrecht** [comment]
relevant code is [here](https://github.com/KotatsuApp/Kotatsu/blob/devel/app%2Fsrc%2Fmain%2Fkotlin%2Forg%2Fkoitharu%2Fkotatsu%2Fsettings%2Fabout%2FAppUpdateDialog.kt#L36-L59):
```
        fun show(version: AppVersion) {
                latestVersion = version
                val message = buildSpannedString {
                        append(activity.getString(R.string.new_version_s, version.name))
                        appendLine()
                        append(activity.getString(R.string.size_s, FileSize.BYTES.format(activity, version.apkSize)))
                        appendLine()
                        appendLine()
                        append(Markwon.create(activity).toMarkdown(version.description))
                }
                MaterialAlertDialogBuilder(activity, DIALOG_THEME_CENTERED)
                        .setTitle(R.string.app_update_available)
                        .setMessage(message)
                        .setIcon(R.drawable.ic_app_update)
                        .setNeutralButton(R.string.open_in_browser) { _, _ ->
                                val intent = Intent(Intent.ACTION_VIEW, version.url.toUri())
                                activity.startActivity(Intent.createChooser(intent, activity.getString(R.string.open_in_browser)))
                        }.setPositiveButton(R.string.update) { _, _ ->
                                downloadUpdate()
                        }.setNegativeButton(android.R.string.cancel, null)
                        .setCancelable(false)
                        .create()
                        .show()
        }
```


I'll go ahead and assume this happens because the text of all three doesn't fit side-by-side.

I suggest these two possible solutions:
- replace "Open in web Browser" by "Changelog" and hope that shortens the label enough
- leave the buttons stacked but ensure all three are visible
